kratos/geometries/geometry.h

```diff
@@ -1337,6 +1337,44 @@ class Geometry : public PointerVector<TPointType>
 
         return rResult;
     }
+    
```

Comment:
i think the doc string is not correct, since the parameters described are not found in the function. I guess that "rPoint is "rCoordinates" (or the contrary)